README.md

```diff
@@ -75,35 +75,23 @@ pip install gh-scoped-creds
 
 ## Client configuration
 
-1. `gh-scoped-creds` uses `git-credentials-store` to provide appropriate authentication,
-    by writing to a `/tmp/gh-scoped-creds` file. This makes sure we don't override
-	the default `~/.git-credentials` file someone might be using. `git` will have to be configured to use
-	the new file.
-
-	You can put the following snippet in `/etc/gitconfig` (for containers) or in
-	`~/.gitconfig`:
-
-	```ini
-	[credential]
-        helper = store --file=/tmp/gh-scoped-creds
-	```
-
-	Or you can run the following command (this puts it in `~/.gitconfig`)
+1. `gh-scoped-creds` will need to know the "Client ID" of the created GitHub app to
+    perform authentication. This can be either set with the environment variable
+	`GH_SCOPED_CREDS_CLIENT_ID`, or be passed in as a commandline parameter `--client-id` to
+	the `gh-scoped-creds` script when users use it to authenticate.
 
-	```
-	git config --global credential.helper "store --file=/tmp/gh-scoped-creds"
-	```
+1. `gh-scoped-creds` uses [`git-credentials-store`](https://git-scm.com/docs/git-credential-store)
+   to provide appropriate authentication, by writing to a `/tmp/gh-scoped-creds`
+   file. This makes sure we don't override the default `~/.git-credentials` file
+   someone might be using. `git` will be automatically configured (via an entry
+   in `~/.gitconfig`) to use this file for github.com credentials.  the new
+   file.
```

Comment:
There is some remnant text here, not sure if its to be deleted or completed.

Reply:
Fixed in https://github.com/yuvipanda/gh-scoped-creds/pull/15! thanks for catching it, @consideRatio